package.json

```diff
@@ -21,35 +21,35 @@
     "history": "^4.10.1",
     "less-plugin-npm-import": "^2.1.0",
     "lodash.throttle": "^4.1.1",
-    "nav-frontend-alertstriper": "^3.0.23",
-    "nav-frontend-alertstriper-style": "^2.0.17",
-    "nav-frontend-chevron": "^1.0.13",
-    "nav-frontend-chevron-style": "^0.3.5",
-    "nav-frontend-core": "^5.0.1",
-    "nav-frontend-ekspanderbartpanel": "^3.0.24",
-    "nav-frontend-ekspanderbartpanel-style": "^1.0.19",
-    "nav-frontend-ikoner-assets": "^2.0.3",
-    "nav-frontend-js-utils": "^1.0.9",
-    "nav-frontend-knapper": "^2.0.9",
-    "nav-frontend-knapper-style": "^1.0.1",
-    "nav-frontend-lenkepanel": "^1.0.47",
-    "nav-frontend-lenkepanel-style": "^0.3.31",
-    "nav-frontend-lenker": "^1.0.41",
-    "nav-frontend-lenker-style": "^0.2.29",
-    "nav-frontend-lesmerpanel": "0.0.42",
-    "nav-frontend-lesmerpanel-style": "0.0.31",
-    "nav-frontend-paneler": "^2.0.10",
-    "nav-frontend-paneler-style": "^0.3.21",
-    "nav-frontend-spinner": "^2.0.5",
-    "nav-frontend-spinner-style": "^0.2.5",
-    "nav-frontend-skjema": "^3.0.18",
-    "nav-frontend-skjema-style": "^2.0.10",
-    "nav-frontend-typografi": "^2.0.26",
-    "nav-frontend-typografi-style": "^1.0.22",
-    "nav-frontend-veileder": "^2.0.21",
-    "nav-frontend-veileder-style": "^2.0.15",
-    "nav-frontend-veilederpanel": "^0.1.29",
-    "nav-frontend-veilederpanel-style": "0.0.29",
+    "nav-frontend-alertstriper": "^3.0.25",
+    "nav-frontend-alertstriper-style": "^2.0.19",
+    "nav-frontend-chevron": "^1.0.15",
+    "nav-frontend-chevron-style": "^0.3.7",
+    "nav-frontend-core": "^5.0.3",
+    "nav-frontend-ekspanderbartpanel": "^3.0.27",
+    "nav-frontend-ekspanderbartpanel-style": "^1.0.21",
+    "nav-frontend-ikoner-assets": "^2.0.5",
+    "nav-frontend-js-utils": "^1.0.11",
+    "nav-frontend-knapper": "^2.0.10",
+    "nav-frontend-knapper-style": "^1.0.2",
+    "nav-frontend-lenkepanel": "^1.0.48",
+    "nav-frontend-lenkepanel-style": "^0.3.32",
+    "nav-frontend-lenker": "^1.0.42",
+    "nav-frontend-lenker-style": "^0.2.30",
+    "nav-frontend-lesmerpanel": "0.0.43",
+    "nav-frontend-lesmerpanel-style": "0.0.32",
+    "nav-frontend-paneler": "^2.0.11",
+    "nav-frontend-paneler-style": "^0.3.22",
+    "nav-frontend-skjema": "^3.0.21",
+    "nav-frontend-skjema-style": "^2.0.13",
+    "nav-frontend-spinner": "^2.0.6",
+    "nav-frontend-spinner-style": "^0.2.6",
+    "nav-frontend-typografi": "^2.0.27",
+    "nav-frontend-typografi-style": "^1.0.23",
+    "nav-frontend-veileder": "^2.0.22",
+    "nav-frontend-veileder-style": "^2.0.16",
+    "nav-frontend-veilederpanel": "^0.1.30",
+    "nav-frontend-veilederpanel-style": "0.0.30",
```

Comment:
Det var mye oppdateringer. Man kunne lagt inn i kalenderen hver tredje måned en påminnelse på at det er på tide å oppgradere frontenbiblioteker.

Reply:
Det kommer nye oppdateringer ganske ofte. Jeg tenker det viktigste her er at vi ikke blir liggende alt for langt bak. Da blir det en større jobb å fikse hvis det blir gjort endringer som ikke er bakoverkompatible. 

En påminnelse i kalenderen kan være fint, men oppgradering av pakker er ikke en oppgave som bør henge på enkeltpersoner. :) Kom gjerne med innspill på hvordan vi kan forankre dette bedre i teamet.